compiler/luci-interpreter/pal/linux/PALArgMax.h

```diff
@@ -26,7 +26,7 @@ static inline void ArgMinMax(const tflite::RuntimeShape &input1_shape, const T1
                              const T2 *axis, const tflite::RuntimeShape &output_shape,
                              T3 *output_data, const std::greater<T1> cmp)
 {
-  tflite::optimized_ops::ArgMinMax(input1_shape, input1_data, axis, output_shape, output_data, cmp);
+  tflite::reference_ops::ArgMinMax(input1_shape, input1_data, axis, output_shape, output_data, cmp);
```

Comment:
For reviewers: in TF 2.3 "optimized" version was actually reference to reference implementation:
``` c++
 63 namespace tflite {
 64 namespace optimized_ops {
 65 
 66 // Unoptimized reference ops:
 67 using reference_ops::ArgMax;
 68 using reference_ops::ArgMinMax;
```